src/components/MoneyRequestReportView/MoneyRequestReportTableHeader.tsx

```diff
@@ -7,12 +7,34 @@ import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
 import type {TranslationPaths} from '@src/languages/types';
 
+type ShouldShowSearchColumnFn = (isIOUReport: boolean) => boolean;
```

Comment:
I don't think this needs to be a named type - let's just inline it in the one place it's used